### PR TITLE
Update project requirements and travis file

### PR DESCRIPTION
**Pull request by Jakeway**
Move dependencies needed for testing into dev_requirements.txt and then call that in the travis configuration file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     url='https://github.com/jb076/django_cache_helper.git',
     packages=find_packages(),
     include_package_data=True,
-    install_requires=['setuptools'],
+    install_requires=['setuptools', 'pylibmc'],
```
Comment (ara818) [inline setup.py]: take out pylibmc.

File: dev_requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+django-nose
```
Comment (ara818) [inline dev_requirements.txt]: add a requirements.txt file, Django >= 1.7

File: .travis.yml [hunk 3]
```diff
@@ -2,13 +2,13 @@ language: python
 python:
     - 3.4
 env:
-    - DJANGO_VERSION=1.8.17
+    - DJANGO_VERSION=1.8.4
+    - DJANGO_VERSION=1.9.13
```
Comment (ara818) [inline .travis.yml]: Add all the djangos 1.7 and up (why, cause ydata uses django 1.7 still :(

**Comment by Jakeway** [comment]
@ara818 ready for another look

File: .travis.yml [hunk 4]
```diff
@@ -2,13 +2,17 @@ language: python
 python:
     - 3.4
 env:
-    - DJANGO_VERSION=1.8.17
+    - DJANGO_VERSION=1.7.10
+    - DJANGO_VERSION=1.8.4
```
Comment (ara818) [inline .travis.yml]: let's do the latest dot release of each

File: dev_requirements.txt (hunk 2 above)
Comment (ara818) [reply dev_requirements.txt]: i kind of hate that we use nose here btw, no need really ... can we create a follow up ticket to move us to regular django tests ... should be very very easy

**Pull request merged**